src/blog/2023/09/making-sense-of-your-industrial-data.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Modernize your legacy industrial data
+subtitle: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+description: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
+date: 2023-09-11
+authors: ["stephen-mclaughlin"]
+image: "/blog/2023/09/images/industrial-legacy-data-blog-image.png"
+tags:
+    - posts
+    - node-red
+    - community
+---
+
+In today's rapidly evolving industrial landscape, the ability to collect, analyse, and act upon data is more critical than ever.
+The Industrial Internet of Things (IIoT) has ushered in a new era of connectivity and automation, but what about the wealth of data locked away in legacy industrial protocols like ModBus?
+How can we unlock and make sense of this valuable information in the IIoT era? That's where tools like Node-RED and `node-red-contrib-buffer-parser` come into play.
```

Comment:
```suggestion
In today's rapidly evolving industrial landscape, the ability to collect, analyse, and act upon data is more critical than ever.
The Industrial Internet of Things (IIoT) has ushered in a new era of connectivity and automation, but what about the wealth of data locked away in legacy industrial protocols like ModBus? How can we unlock and make sense of this valuable information in _any_ data, e.g. serial comms with no protocol, Siemens S7, Omron Fins, Misti MC, or Allen Bradley DF1 
in the IIoT era? That's where tools like Node-RED and `node-red-contrib-buffer-parser` come into play.
```

I liked your explanation on why the title is what it now is, so I copied part over to the intro. Let me know what you think.

Reply:
resolved in [6958872](https://github.com/flowforge/website/pull/1111/commits/6958872243434546368aa0dccea78bf37412a0c9)